File: XMLApp/DTO/NewFlightDTO.cs
using XMLApp.Model;

namespace XMLApp.DTO
{
    public class NewFlightDTO
    {
        public GeoLocation? TakeOffLocation { get; set; }
        public GeoLocation? LandingLocation { get; set; }
        public DateTime? TakeOffDate { get; set; }
        public DateTime? LandingDate { get; set; }
        public int Quantity { get; set; }
        public double Price { get; set; }
    }
}

File: xml-front-app/src/app/model/new-flight-dto.ts
export class NewFlightDTO {
    takeOffLocation: Geolocation = new Geolocation()
    landingLocation: Geolocation = new Geolocation()
    takeOffDate: Date = new Date()
    landingDate: Date = new Date()
    quantity: number = 0
    price: number = 0

    public constructor(obj?: NewFlightDTO){
        if(obj){
            this.takeOffLocation = obj.takeOffLocation
            this.landingLocation = obj.landingLocation
            this.landingDate = obj.landingDate
            this.takeOffDate = obj.takeOffDate
            this.quantity = obj.quantity
            this.price = obj.price
        }
    }
}
